### PR TITLE
Added altcoin payments

### PR DESCRIPTION
**Pull request by DarrenWestwood**
Added altcoin payments and updated to match woocommerce, https://github.com/blockonomics/easy-digital-downloads-plugin/issues/7

### CODE REVIEW

**Comment by blockonomics** [comment]
@jusasiiv  Please revview

**Comment by blockonomics** [comment]
@jusasiiv  on review please also post screenshots of checkout page

File: edd-blockonomics.php [hunk 1]
```diff
@@ -57,10 +57,16 @@ public function __construct()
     add_action( 'init',                         array( $this, 'textdomain' ) );
     add_action( 'edd_gateway_blockonomics',         array( $this, 'process_payment' ) );
     add_action( 'init',                         array( $this, 'listener' ) );
-    //	add_action( 'template_redirect',            array( $this, 'process_failed_payment' ) );
```
Comment (jusasiiv) [inline edd-blockonomics.php]: Why is this code commented out? Either should be removed or comment added why this part is in comments. Keeps the code clean

File: js/app.js [hunk 2]
```diff
@@ -33,73 +74,452 @@ function getParameterByNameBlocko(name, url) {
...
+    }
+
+    //Check if the bitcoin address is present
+    if (typeof $scope.address != 'undefined') {
+        //Fetch the order using address
+        Order.get({
+            "get_order": $scope.address
+        }, function(data) {
+            $scope.order = data;
+            console.log(data);
```
Comment (jusasiiv) [inline js/app.js]: Maybe not log anything to console on prod

**Comment by blockonomics** [comment]
Make sure we use [our referral code](https://github.com/blockonomics/woocommerce-plugin/issues/126) using order API call

File: edd-blockonomics.php (hunk 1 above)
Comment (DarrenWestwood) [reply edd-blockonomics.php]: The commented code was included in the first commit and has been removed now.

File: js/app.js [hunk 3]
```diff
@@ -33,73 +74,451 @@ function getParameterByNameBlocko(name, url) {
...
+            }),
+            new Promise((resolve, reject) => {
+                //Create the new altcoin order
+                AltcoinNew.save({
+                        "order": {
+                            "from_currency": altcoin,
+                            "to_currency": "BTC",
+                            "ordered_amount": amount,
+                            "destination": address,
+                            "referral_code": 'BLOCKO'
```
Comment (DarrenWestwood) [inline js/app.js]: Referral code is being used

**Pull request merged**